src/bin/unit_test_module.c

```diff
@@ -460,13 +460,13 @@ static bool do_xlats(char const *filename, FILE *fp)
 	char		input[8192];
 	char		output[8192];
 	REQUEST		*request;
-	struct timeval	now;
+	fr_time_t 	now;
 
 	/*
 	 *	Create and initialize the new request.
 	 */
 	request = request_alloc(NULL);
-	gettimeofday(&now, NULL);
+	now = fr_time();
```

Comment:
I don't think now is actually used anywhere in the function so you can remove it...